internal/server/proto/server.proto

```diff
@@ -2443,6 +2471,9 @@ message EntrypointConfig {
   // be nil if the user is running an old enough server so always nil-check this.
   DeploymentInfo deployment = 4;
 
+  // The signal to send the application when config files change.
+  string file_change_signal = 6;
```

Comment:
Nitpick: I'd move this line up below the config_sources entry since this is config related. Also consider a `config_` prefix like `config_file_change_signal` or just `config_file_signal`

Reply:
Sure, no prob.